### PR TITLE
Use single bundle with blob web worker

### PR DESCRIPTION
**Pull request by pkerpedjiev**
- [ ] Update the version number in unpkg link in README.md if building a new release

### CODE REVIEW

File: src/PileupTrack.js [hunk 1]
```diff
@@ -192,11 +194,13 @@ const PileupTrack = (HGC, ...args) => {
...
-      const worker = spawn(
-        new Worker('./bam-fetcher-worker.js', {
-          _baseURL: baseUrl,
-        }),
-      );
+      // const worker = spawn(
+      //   new Worker('./bam-fetcher-worker.js', {
+      //     _baseURL: baseUrl,
+      //   }),
+      // );
```
Comment (flekschas) [inline src/PileupTrack.js:201]: Maybe remove? It's in the git history anyway

File: webpack.config.js [hunk 2]
```diff
@@ -6,19 +6,53 @@ const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
...
+class WaitPlugin extends WebpackBeforeBuildPlugin {
+  constructor(file, interval = 100, timeout = 10000) {
+    super(function (stats, callback) {
+      let start = Date.now();
+
+      function poll() {
+        if (fs.existsSync(file)) {
+          callback();
+        } else if (Date.now() - start > timeout) {
+          throw Error("Maybe it just wasn't meant to be.");
```
Comment (flekschas) [inline webpack.config.js:23]: A slightly more informative message could be nice :D

File: webpack.config.js [hunk 3]
```diff
@@ -6,19 +6,53 @@ const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
...
     path: path.resolve(__dirname, 'dist'),
   },
   devServer: {
-    contentBase: [
-      path.join(__dirname, 'node_modules/higlass/dist'),
-    ],
+    contentBase: [path.join(__dirname, 'node_modules/higlass/dist')],
     watchContentBase: true,
+    port: 8077,
+    writeToDisk: true,
```
Comment (flekschas) [inline webpack.config.js:55]: Are these changes necessary?

**Review by flekschas — APPROVED** [review]
Looks great! Thanks for the PR. Just some micro comments.

File: src/PileupTrack.js (hunk 1 above)
Comment (pkerpedjiev) [reply src/PileupTrack.js:201]: Yup, good point. I can also get rid of the `getThisScriptLocation` function.

File: webpack.config.js (hunk 3 above)
Comment (pkerpedjiev) [reply webpack.config.js:55]: The port one isn't. The write to disk is because we need it for the dependency resolution.

File: webpack.config.js (hunk 2 above)
Comment (pkerpedjiev) [reply webpack.config.js:23]: Haha, done!
